helm/cluster-openstack/templates/_openstack_cluster_template.tpl

```diff
@@ -27,4 +27,16 @@ spec:
       - {{ . }}
       {{- end }}
       {{- end }}
+      bastion:
+        enabled: true
+        instance:
+          flavor: {{ .Values.bastion.flavor }}
+          image: {{ .Values.bastion.image }}
+          sshKeyName: locked
+        {{- if .Values.bastion.rootVolume.sourceUUID }}
+          rootVolume:
+            sourceType: image
+            diskSize: {{ .Values.bastion.rootVolume.diskSize }}
+            sourceUUID: {{ .Values.bastion.rootVolume.sourceUUID }}
+        {{- end }}
```

Comment:
It would be good to make sure it's always correct `image`/`sourceUUID`. Otherwise we won't have SSH access. How to achieve that?